### DISCUSSION TITLE
Failure response when deploying using Versions

### DISCUSSION THREAD

**Discussion by zachhutchins** [mia-platform/community#488] [Q&A]
### What product are you having troubles with?

Console

### What Console version are you using?

1.98.6

## Environmental considerations
* Using Advanced workflow
* Using 'Pull' Deployment Option
* Using Extension-api provider

### Steps to reproduce (Error 1)

(Tag does not exist currently in Github)
* Create Version in Mia
* Hit Deploy
* Response -> 500 : failed configuration generation: Tag can be created only from an existing branch

### Steps to reproduce (Error 2)

( Create Tag myself in Github )
* Create Version in Mia
* Hit Deploy
* Response -> 500 : failed configuration generation: Cannot read properties of undefined (reading 'oid')

---

Commit for the configuration still makes it to Github.

**Comment by fredmaggiowski** [comment]
Hi @zachhutchins, thanks for reaching out!

We are sorry to hear you are having this kind of issue, it seems like an integration bug with GitHub. Please open a ticket to your Mia-Platform referent so we can further investigate it and resolve it asap
